bin/evaluate-measurements.js

```diff
@@ -97,27 +97,28 @@ async function processRound (roundIndex, measurements, resultCounts) {
     prepareProviderRetrievalResultStats: async () => {}
   })
 
+  if (!keepRejected) {
+    round.measurements = round.measurements.filter(m => m.taskingEvaluation === 'OK')
+  }
+
   for (const m of round.measurements) {
-    // FIXME: we should include non-majority measurements too
-    // See https://github.com/filecoin-station/spark-evaluate/pull/396
-    if (m.taskingEvaluation !== 'OK' && m.consensusEvaluation === 'MAJORITY_RESULT') continue
     resultCounts.total++
-    resultCounts[m.retrievalResult] = (resultCounts[m.retrievalResult] ?? 0) + 1
-  }
 
-  if (!keepRejected) {
-    round.measurements = round.measurements
-      // Keep accepted measurements only
-      // FIXME: we should include non-majority measurements too
-      // See https://github.com/filecoin-station/spark-evaluate/pull/396
-      .filter(m => m.taskingEvaluation === 'OK' && m.consensusEvaluation === 'MAJORITY_RESULT')
-      // Remove the taskingEvaluation and consensusEvaluation fields as all accepted measurements have the same value
-      .map(m => ({ ...m, taskingEvaluation: undefined, majorityEvaluation: undefined }))
+    let status
+    if (m.taskingEvaluation !== 'OK') {
+      status = m.taskingEvaluation
+    } else if (m.consensusEvaluation !== 'MAJORITY_RESULT') {
+      status = m.consensusEvaluation
+    } else {
+      status = m.retrievalResult
+    }
+
+    resultCounts[status] = (resultCounts[status] ?? 0) + 1
```

Comment:
After I posted my comment above and re-read your code, I realised the version presented here is a great way how to summarise the measurements when `KEEP_REJECTED=1` 👍🏻